opencontrail/files/3.0/collector/contrail-alarm-gen.ini

```diff
@@ -0,0 +1,13 @@
+[program:contrail-alarm-gen]
+command=/usr/bin/contrail-alarm-gen -c /etc/contrail/contrail-alarm-gen.conf
+priority=440
+autostart=true
+killasgroup=true
+stopsignal=KILL
```

Comment:
Don't know, but that was there originally (based on what came out of the package)